module-1/lab-data_cleaning/your-code/main.ipynb

```diff
@@ -9,10 +9,12 @@
   },
```

Comment:
1. Step 7: You can change column names this way but you don't need to type all collumns in your dataframe. You have choose the most inefficient way of doing it. In further questions, you began to use the rename method, this is an optimized option. 
2. When you need to print the output of a chunk of code involving a dataframe it's a best practice to use the head() method to avoid "printing" all the dataframe when it's not necessary. 
3. Steps 9,10 and 11 are not done, are you sure this is your last main file version? Get back to me once you have checked it. Thanks!!